### PR TITLE
Better documentation for user@ services

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/user@.service.xml [hunk 1]
```diff
@@ -17,34 +17,37 @@
...
-    <filename>user@<replaceable>UID</replaceable>.service</filename>, where the user's numerical UID
-    is used as the instance identifier. Each <command>systemd --user</command> instance manages a
-    hierarchy of its own units. See
-    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry> for
-    a discussion of systemd units and
-    <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    for a list of units that form the basis of the unit hierarchies of system and user units.</para>
+    <filename>user@<replaceable>UID</replaceable>.service</filename>, with the user's numerical UID used as
+    the instance identifier. Those instances use the same executable as the system manager, but running in a
+    mode where it starts a different set of units specific. Each <command>systemd --user</command> instance
```
Comment (poettering) [inline man/user@.service.xml]: `… of units specific`? Is that a valid expression?

File: man/user@.service.xml [hunk 2]
```diff
@@ -17,34 +17,37 @@
...
+    manages a hierarchy of its own units to that user. See
```
Comment (daandemeyer) [inline man/user@.service.xml]: `to that user` => `specific to that user`?

File: man/user@.service.xml (hunk 1 above)
Comment (keszybz) [reply man/user@.service.xml]: Not, I messed something up.

**Comment by keszybz** [comment]
Thanks for the reviews. I fixed the grammar.

**Comment by keszybz** [comment]
(I merged this without waiting for CI because the CI passed last time, and now there were only textual changes.)

**Pull request merged**